src/main/java/com/aws/greengrass/mqtt/bridge/model/MqttMessage.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.mqtt.bridge.model;
+
+import lombok.NonNull;
+import lombok.Value;
+
+// TODO support mqtt5
+@Value
+public class MqttMessage implements Message {
```

Comment:
Rename to LocalMqttMessage

Reply:
Was also using this in iot core client, did we want a separate type for that?